.github/workflows/compile-rtk-everywhere.yml

```diff
@@ -103,6 +109,11 @@ jobs:
           cp libmbedcrypto.a /home/runner/.arduino15/packages/esp32/tools/esp32-arduino-libs/${{ env.ESP_IDF }}/esp32/lib/libmbedcrypto.a
           cp libmbedx509.a /home/runner/.arduino15/packages/esp32/tools/esp32-arduino-libs/${{ env.ESP_IDF }}/esp32/lib/libmbedx509.a
 
+      - name: Patch NetworkEvents
```

Comment:
Add comment that says this patch is needed until ESP vx.x.x and maybe link to issue on core repo.